.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+<!--
+Allo' allo'! 
```

Comment:
[Praise] I like this little bit of delighter text 😄 very approachable and friendly.

Reply:
+1 Allo' Allo' !